po/R-ca.po

```diff
@@ -0,0 +1,293 @@
+msgid "Content not found. Please use links in the navbar."
+msgstr ""
+"No s'ha trobat el contingut. Useu els enllaços de la barra de navegació."
+
+#: build-articles.R:381 navbar.R:136 navbar.R:143 navbar.R:158
+msgid "Articles"
+msgstr "Articles"
+
+#: build-articles.R:430
+msgid "All vignettes"
+msgstr "Totes les vinyetes"
+
+#: build-footer.R:23
+msgid "Developed by"
+msgstr "Desenvolupat per"
+
+#: build-footer.R:28
+msgid "Site built with <a href=\"%s\">pkgdown</a> %s."
+msgstr "Lloc construït amb <a href=\"%s\">pkgdown</a> %s."
+
+#: build-home-authors.R:69
+msgid "More about authors..."
+msgstr "Més sobre els autors..."
+
+#: build-home-authors.R:72
+msgid "Developers"
+msgstr "Desenvolupadors"
+
+#: build-home-authors.R:77 render.R:96
+msgid "Authors"
+msgstr "Autors"
+
+#: build-home-authors.R:169
+msgid "author"
+msgstr "autor"
+
+#: build-home-authors.R:170
+msgid "compiler"
+msgstr "compilador"
+
+#: build-home-authors.R:171
+msgid "contractor"
+msgstr "contractista"
+
+#: build-home-authors.R:172
+msgid "contributor"
+msgstr "col·laborador"
+
+#: build-home-authors.R:173
+msgid "copyright holder"
+msgstr "titular dels drets d'autor"
+
+#: build-home-authors.R:174
+msgid "maintainer"
+msgstr "mantenidor"
+
+#: build-home-authors.R:175
+msgid "data contributor"
+msgstr "col·laborador de dades"
+
+#: build-home-authors.R:176
+msgid "funder"
+msgstr "patrocinador"
+
+#: build-home-authors.R:177
+msgid "reviewer"
+msgstr "revisor"
+
+#: build-home-authors.R:178
+msgid "thesis advisor"
+msgstr "supervisor de tesi"
+
+#: build-home-authors.R:179
+msgid "translator"
+msgstr "traductor"
+
+#: build-home-citation.R:36
+msgid "Citing %s"
+msgstr "Com citar %s"
+
+#: build-home-citation.R:91
+msgid "Authors and Citation"
+msgstr "Autors i citació"
+
+#: build-home-community.R:23
+msgid "Contributing guide"
+msgstr "Guia per contribuir"
+
+#: build-home-community.R:27
+msgid "Code of conduct"
+msgstr "Codi de conducta"
+
+#: build-home-community.R:31
+msgid "Getting help"
+msgstr "Obtenció d'ajuda"
+
+#: build-home-community.R:38
+msgid "Community"
+msgstr "Comunitat"
+
+#: build-home-index.R:88
+msgid "Dev Status"
+msgstr "Estat de desenvolupament"
+
+#: build-home-index.R:154
+msgid "View on %s"
+msgstr "Mostra a %s"
+
+#: build-home-index.R:155
+msgid "Browse source code"
+msgstr "Navega pel codi font"
+
+#: build-home-index.R:156
+msgid "Report a bug"
+msgstr "Informa d'un error"
+
+#: build-home-index.R:160
+msgid "Links"
+msgstr "Enllaços"
+
+#: build-home-index.R:165
+msgid "Table of contents"
+msgstr "Taula de continguts"
+
+#: build-home-license.R:12 build-home-license.R:34
+msgid "License"
+msgstr "Llicència"
+
+#: build-home-license.R:29
+msgid "Full license"
+msgstr "Llicència completa"
+
+#: build-news.R:101 build-news.R:246 build-news.R:251
+msgid "Changelog"
+msgstr "Registre de canvis"
+
+#: build-news.R:126
+msgid "Version %s"
+msgstr "Versió %s"
+
+#: build-news.R:138 build-news.R:240
+msgid "News"
+msgstr "Notícies"
```

Comment:
It seems closer to Novetats, but Notícies might be a good literal translation too. I checked with [softcatalà resources](https://static.softcatala.org/terminology/tots-glossary.html) and it seems more common to translate for Notícies

Reply:
Ok, thanks! I better like your proposal of «Novetats».